tests/test_memory_leak.py

```diff
@@ -127,7 +127,7 @@ def epochs(self) -> int:
 
     @property
     def max_memory_threshold_mb(self) -> float:
-        return 1600
+        return 2200
```

Comment:
How did you decide on the new float thresholds?

Reply:
Just increased them based on the information from several CI runs. + did the memory leak investigation locally (to make sure there is no memory leak)